### ISSUE TITLE
Add a high contrast mode

### ISSUE THREAD

**Issue by steffenkleinle** [digitalfabrik/integreat-app#2781]
**Is your feature request related to a problem? Please describe.**
As a visually impaired user I want to be able to use the app. For this reason we want to introduce a new high contrast mode with maximum contrast.

**Describe the solution you'd like**
Decide which places a high contrast mode changes.
Add a high contrast mode in the theme/code (to be discussed how).

Recoloring the 1st level icons (#2780) and toggling the high contrast mode (#2778) are NOT part of this issue.

**Describe alternatives you've considered**
None.

**Additional context**
Blocks: #2778.
Relates to #2778, #2780.
Part of #1021.

**Comment by ztefanie** [comment]
This not merged PR contains the changes to get information of high contrast mode settings in android: 

https://github.com/digitalfabrik/integreat-app/pull/2795
